### PR TITLE
Added Traffic Recognition

### LINKED ISSUE #331: A pretrained model for autonomous traffic sign detection
# Aim

- With the growing popularity in DL/ML, I believe having some Deep Learning projects would be a good idea.

# Details

- I propose a pre-trained model, of traffic sign detection, simple enough so that even a beginner can understand, but flexible enough so that a hobbyist can easily optimize the hyperplanes for a more structured prediction and detection

### PR DESCRIPTION
**Pull request by vybhav72954**
# Description

Below is a Trained Model, along with a training regime based on a CNN centered multilayer model.
One can find the training details, all the models, and the metrics.
the highest accuracy achieved was 99.7%.

## Fixes #331 

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by avinashkranjan — APPROVED** [review]
LGTM..🔥
